fraud-detection-update-sample/tests/test_main.py

```diff
@@ -7,10 +7,13 @@
 parent_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
 sys.path.append(parent_dir)
 from main import *
+# Get function environment variable for GCP project ID to use for accessing Earth Engine.
+GCP_PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "river-sky-386919")
+
 
 class TestMain(unittest.TestCase):
     def test_parse_path_parts(self):
-        firestore_payload_name = "projects/river-sky-386919/databases/(default)/documents/untrusted_samples/fd1239d5488215a23440"
+        firestore_payload_name = "projects/{GCP_PROJECT_ID}/databases/(default)/documents/untrusted_samples/fd1239d5488215a23440"
```

Comment:
we need better unit tests yes?  isnt this just testing parse_path_parts?

Reply:
yes .. @jjcastro or @VimWizardVersace  wrote this down